specification/postgresqlhsc/resource-manager/Microsoft.DBforPostgreSQL/preview/2023-03-02-preview/postgresqlhsc.json

```diff
@@ -2610,6 +2647,26 @@
         }
       }
     },
+    "IdentityProperties": {
+      "type": "object",
+      "description": "Describes the identity of the cluster.",
+      "properties": {
+        "type": {
+          "type": "string",
+          "enum": [
```

Comment:
>enum [](http://example.com/codeflow?start=11&length=4)

Do you not also allow both system and user assigned together? Many resource types do.

Hence the definition in managedidentity.json

```
"ManagedServiceIdentityType": {
      "description": "Type of managed service identity (where both SystemAssigned and UserAssigned types are allowed).",
      "enum": [
        "None",
        "SystemAssigned",
        "UserAssigned",
        "SystemAssigned,UserAssigned"
      ],
      "type": "string",
      "x-ms-enum": {
        "name": "ManagedServiceIdentityType",
        "modelAsString": true
      }
    },

Reply:
I don't think there are plans to add this, but I will open the topic for discussion with the developers of this feature.